geode-concurrency-test/src/main/java/org/apache/geode/test/concurrency/loop/LoopRunner.java

```diff
@@ -35,7 +35,7 @@
  * Simple runner that just runs the test in a loop
  */
 public class LoopRunner implements Runner {
-  private static final int DEFAULT_COUNT = 1000;
+  private static final int DEFAULT_COUNT = 2000;
```

Comment:
Just curious - what prompted this change?  The fix for the intermittent test failure is recategorizing the test as an integration test instead of unit test (the change in the build.gradle) right?  Was this also just something you recognized as an improvement so you wanted to get this in as well?  Or did you encounter something while looking into this that instigated this change?

Reply:
No, this is what makes the test more reliable. The move was a change I wanted to make because unit tests should not be flaky and this test is by its nature.
The test in question is testing that a concurrency runner for tests actually catches issues due to parallelism. It was flaky because the test expects an assertion failure that is caused by timing, and this timing was not always occurring within 1000 iterations on JDK11. Increasing the number of iterations will increase the pass rate of the test, but will still not guarantee that it passes every time. I chose to double it because I did not want to make the tests that use the LoopRunner all take x10 (for example) longer to run by default, and x2 seems reasonable for more confidence that it will catch issues.

Reply:
That makes sense.  I should have looked closer - I had assumed the recategorization of the test type was due to pollution of global state, because I've seen that sort of recategorization done before for that reason.  If we see this fail again with the new 2000 iterations, then do you suppose we can just increase it again if necessary?  I suppose it is tricky with the nondeterministic nature of what its testing.  Either way, thanks for clarifying.